pkg/localdata/profile.go

```diff
@@ -321,6 +328,60 @@ func (p *Profile) SelectInstalledVersion(component string, version v0manifest.Ve
 	return version, nil
 }
 
+// ResetMirror reset root.json and cleanup manifests directory
+func (p *Profile) ResetMirror(addr, root string, forceRefresh bool) error {
```

Comment:
Should we convert the `addr` to absolute path if the current addr is an offline mirror?

Reply:
We can early return if the mirror address is not changed.